liferay-idea-plugin/README.markdown

```diff
@@ -1,9 +1,88 @@
-Run in command line by
-./gradlew runIde
-or
-Run in IntelliJ IDEA by
-1. Install the latest IntelliJ IDEA community.
-2. Import this folder as Gradle Layout.
-3. Config Java SDK and InterlliJ Plugin SDK.
-4. Add gradle configuration with project intellij and task runIde. Save
-5. Click run or debug.
\ No newline at end of file
+# Liferay Idea Plugin
+
+## Quick Start
+You can download the plugin zip file from:
+
+```
+https://release.liferay.com/1/userContent/liferay-idea-plugin/0.x/
```

Comment:
We will need to put this in a temporary location  since this is protected.  Maybe bintray? or as a binary file on github?

Reply:
Yes, I think so.
How can we put the binary file on github?